### DISCUSSION TITLE
Add Non_Negative_Difference to Calculation and to Reduce Transformation

### DISCUSSION THREAD

**Discussion by UlrichThiess** [grafana/grafana#63310] [Dashboards, Panels & Visualizations]
Some times a growing counter wraps so "Difference" calculates with negative Values.

Please add "Difference NonNegative" to the Calculation:
![grafik](https://user-images.githubusercontent.com/79257887/218426552-376b58c0-aba2-43c7-826c-d059103add62.png)

and to the Reduce function in the Transformations:
![grafik](https://user-images.githubusercontent.com/79257887/218426724-00be21ae-3103-426a-8ad6-6594a05ec41d.png)

Thanks.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82854. Feel free to continue the discussion around this there. Thank you!
